site/pages/create-disruption/[disruptionId].page.tsx

```diff
@@ -437,78 +498,22 @@ const CreateDisruption = (props: DisruptionPageProps): ReactElement => {
-                            hint="Enter the time in 24hr format. For example 0900 is 9am, 1730 is 5:30pm"
-                            value={pageState.inputs.publishStartTime}
-                            disabled={false}
-                            inputName="publishStartTime"
-                            stateUpdater={stateUpdater}
-                            initialErrors={pageState.errors}
-                            schema={createDisruptionSchema.shape.publishStartTime}
-                        />
+                    <input type="hidden" name="disruptionId" value={props.disruptionId} />
 
-                        <DateSelector<DisruptionInfo>
-                            display="End date"
-                            hiddenHint="Enter in format DD/MM/YYYY"
-                            value={pageState.inputs.publishEndDate}
-                            disabled={validity.disruptionNoEndDateTime === "true"}
-                            disablePast={false}
-                            inputName="publishEndDate"
-                            stateUpdater={stateUpdater}
-                            initialErrors={pageState.errors}
-                            schema={createDisruptionSchema.shape.publishEndDate}
-                        />
-
-                        <TimeSelector<DisruptionInfo>
-                            display="End time"
-                            hint="Enter the time in 24hr format. For example 0900 is 9am, 1730 is 5:30pm"
-                            value={pageState.inputs.publishEndTime}
-                            disabled={validity.disruptionNoEndDateTime === "true"}
-                            inputName="publishEndTime"
-                            stateUpdater={stateUpdater}
-                            initialErrors={pageState.errors}
-                            schema={createDisruptionSchema.shape.publishEndTime}
-                        />
-
-                        <input type="hidden" name="disruptionId" value={props.disruptionId} />
+                    <button className="govuk-button" data-module="govuk-button">
+                        Save and continue
+                    </button>
 
-                        <button className="govuk-button mt-8" data-module="govuk-button">
-                            Save and continue
-                        </button>
-
-                        {displayCancelButton && pageState.disruptionId ? (
-                            <Link
-                                role="button"
-                                href={`${queryParams["return"] as string}/${pageState.disruptionId}`}
-                                className="govuk-button mt-8 ml-5 govuk-button--secondary"
-                            >
-                                Cancel Changes
-                            </Link>
-                        ) : null}
-                        <button
+                    {displayCancelButton && pageState.disruptionId ? (
+                        <Link
```

Comment:
The cancel button does not align with Save button

Reply:
Fixed